includes/wf_crm_webform_postprocess.inc

```diff
@@ -1960,12 +1960,23 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
     $this->saveCustomData($this->data['contribution'][1], $id, 'Contribution', FALSE);
     // Save soft credits
     if (!empty($contribution['soft'])) {
+      // Get total amount from total amount after line item calculation
+      $amount = $this->totalContribution;
+      
+      // Get Default softcredit type
+      $default_soft_credit_type = wf_civicrm_api('OptionValue', 'getsingle', array(
+        'return' => "value",
+        'option_group_id' => "soft_credit_type",
+        'is_default' => 1,
+      ));
+
```

Comment:
Default ("Solicited" unless configured by admins) is ok for now - though there are 11 options in CiviCRM native - key purpose of this PR is to fix the $0.00 amount;